e2e/cypress/integration/api/iam/05_projects_api_compliance_ingestion.ts

```diff
@@ -0,0 +1,317 @@
+import { describeIfIAMV2p1 } from '../../constants';
+
+describeIfIAMV2p1('Compliance Ingestion project tagging', () => {
+  const cypressPrefix = 'test-projects-api-compliance';
+
+  const projectsWithRule = [
+    // This test is commented out because there is a current limit of 6 projects allowed
```

Comment:
I thought we expanded the project limit for our test envs . . . https://github.com/chef/automate/pull/2051

Where was it failing?

Reply:
I have to test it, but I assumed it would fail in buildkite.

Reply:
Buildkite does only allow 6 projects currently. https://buildkite.com/chef/chef-automate-master-verify-private/builds/7444#0afa3198-03aa-47ff-8f1b-a7b7d315f0bd. 
@bcmdarroch did provide the solution which I will work on in a later PR.